airflow/www_rbac/views.py

```diff
@@ -3282,6 +3284,68 @@ def imprt(self, import_paths, tar):
 
         return self.response(200, message="OK")
 
+    def set_config(self,pathname):
+        config = 'model_config_list: {'
+        dirs = [name for name in os.listdir(pathname) if os.path.isdir(os.path.join(pathname, name))]
+        for dname in dirs:
+            config += '''config: {
+                            name: "%s",
+                            base_path: "/usr/local/couture/trained-models/%s/%s",
+                            model_platform: "tensorflow"
+                        },''' % (dname, 'tf-models', dname)
+        config += '}'
+        try:
+            with open(os.path.join(pathname, 'models.config'), 'w') as f:
```

Comment:
Please keep this in a separate class based view.

Reply:
Done